packages/algorand-js/src/interpreter/parser.ts

```diff
@@ -0,0 +1,154 @@
+import fs from "fs";
+import readline from "readline";
+
+import { TealError } from "../errors/errors";
+import { ERRORS } from "../errors/errors-list";
+import { Operator } from "../types";
+import { Add, Addr, Div, Int, Mul, Pragma, Sub } from "./opcode-list"; ;
+
+var opCodeMap: {[key: string]: any } = {
+  // Pragma
+  "#pragma": Pragma,
+
+  // Pseudo-Ops
+  addr: Addr,
+  int: Int,
+
+  // Arithmetic ops
+  "+": Add,
+  "-": Sub,
+  "/": Div,
+  "*": Mul
+};
+
+/**
+ * Description: Read line and split it into fields
+ * - ignore comments, keep only part that is relevant to interpreter
+ * @param line : Line read from TEAL file
+ */
+/* eslint-disable sonarjs/cognitive-complexity */
+export function fieldsFromLine (line: string): string[] {
```

Comment:
In parsing context we use `word`, `lexem` or `token`. Let's don't use `fields`. We can rename to `wordsFromLine`

Reply:
yes